demand_energy.md

```diff
@@ -106,6 +106,40 @@ The non-fuel costs are estimated for some technologies (e.g., light-duty vehicle
 
 
 
+### Direct Air Capture for Carbon Dioxide Removal
+
+GCAM has the capability to model three technologies which consume thermal and/or electrical energy for the sole purpose of removing carbon dioxide from the atmosphere, broadly referred to as direct air capture with carbon storage (DACCS). The first is an aqueous hydroxide solvent process requiring water, high-temperature thermal energy from natural gas for solvent regeneration, and electricity to run ancillary equipment . The second archetype relies on the same aqueous hydroxide solvent process but uses electricity to generate the high temperatures (>900 ˚C) required for regeneration. The third uses solid adsorbents, with the lower-temperature heat required for sorbent regeneration generated by an electric heat pump. These technologies remove CO2 from the atmosphere and send it to geologic storage, along with any captured combustion emissions from process heat.  
+
+In GCAM, the DACCS technologies indirectly compete with (a) emissions abatement; and (b) other carbon removal technologies such as BECCS and afforestation based on its cost and the subsidy paid for CO2 removal (i.e., the carbon emissions price). This competition is created by defining a fourth "no-DAC" technology which does not capture carbon and has zero cost. We use GCAM’s (unmodified) logit choice model for economic choice between DACCS technologies. This includes the “choice” to not deploy DACCS and instead use other mitigation or negative emissions technologies (i.e., the "no-DAC" technology). The share si of any DACCS technology with price pi is computed as follows:
+
+
+$$
+s_i=\frac{α_i * exp⁡(β*p_i)}{\sum_{j=1}^{N}α_j * exp(β*p_j))}
+$$
+
+
+Where:
+
+$$α_i$$ = the shareweight of the technology. 
+
+$$β$$ = the logit coefficient, which determines how large a cost difference is required to produce a given difference in market share. 
+
+Shareweights are used to represent societal preferences, infrastructure buildup, and barriers or accelerants to market entry. Consistent with GCAM’s treatment of other new and emerging technologies, we set shareweights for DACCS technologies to zero in 2020, and linearly increase to 1 by 2050 for most scenarios. This means that by 2100, DACCS technologies are competing solely based on their cost minus the subsidy for removing carbon dioxide from the atmosphere (again, equal to the carbon emissions price). 
+
+In each region, the base year service demand (i.e., the sum of “no DAC” + “DAC” technologies for which the share of each technology is computed) is set for the USA region at an arbitrarily-high value of 2000 MtC per year, with the deployment share of DAC technologies being determined by the sum of their energy, water, and non-energy costs, minus any carbon price subsidy.  Base year service demand for other regions is scaled linearly by each regions cumulative onshore carbon storage capacity relative to the USA region.
```

Comment:
Is the 2000 MtC truly arbitrary? Wouldn't this impact the results? (Its documenting what's in there, so that's fine, but if there's more explanation for this that would be useful.)

Reply:
Just updated the PR to clarify this